src/main/java/raisetech/Student/Management/controller/StudentController.java

```diff
@@ -12,7 +12,7 @@
 import raisetech.Student.Management.domain.StudentDetail;
 import raisetech.Student.Management.service.StudentService;
 
-@RestController // 修正: @RestController → @Controller
+@Controller // 修正: @RestController → @Controller
```

Comment:
Gitはコミットから過去の修正履歴が終えるのでコメントで変更履歴を残す必要はないですね！
```suggestion
@Controller
```